src/main.rs

```diff
@@ -0,0 +1,28 @@
+use lc3_vm::*;
+use std::env;
+use termios::*;
+pub mod hardware;
+mod lc3_vm;
+
+fn main() -> Result<(), VMError> {
+    let args: Vec<String> = env::args().collect();
```

Comment:
You can use [Clap](https://crates.io/crates/clap) to manage the command line arguments